### PR TITLE
iam controls for foundattional security

### PR DESCRIPTION
**Pull request by amanenk**

### CODE REVIEW

**Review by yevgenypats — APPROVED** [review]

File: queries/iam/policies_with_admin_rights.sql [hunk 1]
```diff
@@ -0,0 +1,17 @@
+WITH policy_statements AS (SELECT p.cq_id AS cq_id, JSONB_ARRAY_ELEMENTS(v.document -> 'Statement') AS statement
+                           FROM aws_iam_policies p
+                                    LEFT JOIN aws_iam_policy_versions v
+                                              ON p.cq_id = v.policy_cq_id AND p.default_version_id = v.version_id),
+     allow_all_statements AS (
+         SELECT cq_id, COUNT(statement) AS statements_count
+         FROM policy_statements
+         WHERE statement ->> 'Action' = '*'
+           AND statement ->> 'Effect' = 'Allow'
+           AND statement ->> 'Resource' = '*'
```
Comment (bbernays) [inline queries/iam/policies_with_admin_rights.sql:10]: What happens if resource is an array?

File: queries/iam/policies_with_admin_rights.sql [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+                                              ON p.cq_id = v.policy_cq_id AND p.default_version_id = v.version_id
+                           WHERE p.arn NOT LIKE 'arn:aws:iam::aws:policy%')
+
+   , allow_all_statements AS (
+    SELECT cq_id, COUNT(STATEMENT) AS statements_count
+    FROM policy_statements
+    WHERE STATEMENT ->> 'Action' = '*'
+      AND STATEMENT ->> 'Effect' = 'Allow'
+      AND (STATEMENT ->> 'Resource' = '*'
+        OR STATEMENT ->> 'Resource' LIKE '%"*"%')
```
Comment (bbernays) [inline queries/iam/policies_with_admin_rights.sql:13]: Does the `LIKE` operator handle JSON arrays?

Comment (amanenk) [reply queries/iam/policies_with_admin_rights.sql:13]: `->>` operator turns everything into string so we can use `like` on it.

Comment (amanenk) [reply queries/iam/policies_with_admin_rights.sql:13]: for example the result of STATEMENT ->> 'Resource' will be string of  `["*"]` and we can use like in this case

**Pull request merged**